beacons/src/commonTest/kotlin/BeaconFlowTest.kt

```diff
@@ -18,7 +18,7 @@
 package com.splendo.kaluga.bluetooth.beacons
 
 import com.splendo.kaluga.bluetooth.device.Device
-import com.splendo.kaluga.test.SimpleFlowTest
+import com.splendo.kaluga.test.base.SimpleFlowTest
```

Comment:
Looking at this, this will cause a lot of downstream pain.

I think we should at least have a (deprecated) typealias for the most common classes that are changing names. Maybe just in`test-utils` (which itself could be removed ~ 1.0)

Reply:
Added a bunch of typealiases / child classes to deal with this. Not all of them might work out of the box (and I still removed the legacy mocking stuff) but should ease migration by a lot